akka-actor-typed-tests/src/test/scala/akka/actor/typed/OrElseStubbedSpec.scala

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (C) 2017-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed
+
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.testkit.typed.scaladsl._
+import org.scalatest.{ Matchers, WordSpec, WordSpecLike }
+
+object OrElseStubbedSpec {
```

Comment:
since this file also contains `OrElseSpec` I think the file rename should be reverted